File: app/src/main/java/com/v2/desafionubank/ui/view/BaseViewHolder.java
package com.v2.desafionubank.ui.view;

import android.support.v7.widget.RecyclerView;
import android.view.View;

import butterknife.ButterKnife;

/**
 * Created by CaioSChristino on 30/03/18.
 */

public class BaseViewHolder extends RecyclerView.ViewHolder {

    public BaseViewHolder(View view) {
        super(view);
        ButterKnife.bind(this, view);
    }
}

File: app/src/main/java/com/v2/desafionubank/ui/view/BaseViewHolder.kt
package com.v2.desafionubank.ui.view

import android.support.v7.widget.RecyclerView
import android.view.View
import butterknife.ButterKnife

/**
 * Created by csanchez on 19/04/2018.
 */

open class BaseViewHolder(view: View) : RecyclerView.ViewHolder(view) {

    init {
        ButterKnife.bind(this, view)
    }
}
